### ISSUE TITLE
Partially transparent layer doesn't render properly in inventory window

### ISSUE THREAD

**Issue by thatoneguydan** [tr7zw/3d-Skin-Layers#68]
As you can see here (in the glasses): https://imgur.com/a/m76gzF4

The partial transparency doesn't render properly inside the inventory layer. It renders correctly in F5 view.

**Comment by tr7zw** [comment]
Hm could be some side effect of the shader potentially? The inventory uses the same rendering as in world, so might be due to some mojank issue.

**Comment by thatoneguydan** [comment]
Hmmm. The issue remains when I deactivate shaders, but maybe it's something to do with iris or sodium.

**Comment by tr7zw** [comment]
Seems to be some issue with Iris on 1.19. I'll contact the iris devs later about that, currently having other issues with it too.

**Comment by tr7zw** [comment]
I think I got these issues fixed recently by using the correct shader to render.

**Comment by thatoneguydan** [comment]
![2024-01-15_11 26 22](https://github.com/tr7zw/3d-Skin-Layers/assets/86075779/b1dd5cf7-a640-4e11-b0f8-399d7a33d060)
![2024-01-15_11 26 33](https://github.com/tr7zw/3d-Skin-Layers/assets/86075779/9cafce8d-c8c1-482d-aa0e-143846d1d29a)

Unfortunately, the issue persists. The glasses are partially transparent and they appear correct when facing head on, but when turning to the side they start to disappear. It appears the same way outside of the inventory window to other players.

**Comment by tr7zw** [comment]
Then there is probably nothing I can do about that, its just transparency being transparency. It is possible that future Sodium or Iris updates might fix that. You could also try turning off the fast render mode in the settings.

**Comment by thatoneguydan** [comment]
Turns out the problem is incompatibility with [Immediately Fast](https://modrinth.com/mod/immediatelyfast).

Everything renders properly with Sodium, Indium, Iris, and 3D Skin Layers installed, as long as Immediately Fast isn't installed.

I tested again with only 3D Skin Layers, Fabric API, and Immediately Fast, and the bug returns. With only Immediately Fast installed and no other mods, everything renders properly.

I opened an issue with Immediately Fast, but I thought I'd keep you in the loop in case you wanted to investigate (since I'm not sure which mod is having incompatibility issues).
